src/getters/crunchbase.py

```diff
@@ -0,0 +1,114 @@
+"""
+discovery_utils.src.getters.crunchbase.py
+
+Module for easy access to downloaded CB data on S3.
+
+"""
+import pandas as pd
+import boto3
+import logging
+import re
+
+
+# Define global variables
+BUCKET_NAME = "discovery-iss"
+S3_PATH = "data/crunchbase/"
+AWS_ACCESS_KEY =
+AWS_SECRET_KEY = 
+FILE_NAMES = [
+    "acquisitions.csv",
+    "category_groups.csv",
+    "checksum.csv",
+    "degrees.csv",
+    "event_appearances.csv",
+    "events.csv",
+    "funding_rounds.csv",
+    "funds.csv",
+    "investment_partners.csv",
+    "investments.csv",
+    "investors.csv",
+    "ipos.csv",
+    "jobs.csv",
+    "org_parents.csv",
+    "organization_descriptions.csv",
+    "organizations.csv",
+    "people_descriptions.csv",
+    "people.csv"
+]
```

Comment:
These will soon use `.parquet`

Reply:
I can think of a couple of ways of handling this
- iterating through the file names (with "." and everything after it removed) to find the exact match
- putting two "try"s in to handle .csv or .parquet

I'd like to keep the logic simple and clean though and these make the function longer.

Reply:
or, append the desired file extension at the end